Revit_oM/Revit_oM.csproj

```diff
@@ -38,6 +38,11 @@
     <Reference Include="BHoM">
       <HintPath>..\..\BHoM\Build\BHoM.dll</HintPath>
     </Reference>
+    <Reference Include="Data_oM, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Reference should not contain version, culture, or processor architecture

Reply:
Fixed